frontend/pom.xml

```diff
@@ -17,7 +17,7 @@
     <web.target>${project.build.directory}/web</web.target>
     <bundle.output>${project.build.directory}/build</bundle.output>
     <bundle.dest>${bundle.output}/META-INF/resources</bundle.dest>
-    <node.install.directory>${project.build.directory}</node.install.directory>
+    <node.install.directory>${download.dir}/zanata-frontend/node-${node.version}-npm-${npm.version}</node.install.directory>
```

Comment:
So, it should be possible for the developer to override this with `-Dnode.install.directory`?  If so, good, but perhaps it should be documented in a readme somewhere (since it is taking a minor liberty with the dev's file system).